lib/airbrake/rake/tasks.rb

```diff
@@ -92,7 +92,7 @@
     end
 
     heroku_env = config.each_line.with_object({}) do |line, h|
-      h.merge!(Hash[*line.rstrip.split('=')])
+      h.merge!(Hash[*line.rstrip.split.flat_map { |v| v.split('=', 2) }])
```

Comment:
The test should be simple enough to add to https://github.com/airbrake/airbrake/blob/master/spec/unit/rake/tasks_spec.rb, and it will make this code much easier to understand and work with.